src/main/flight/pid.c

```diff
@@ -90,13 +90,11 @@ pt1Filter_t throttleLpf;
 
 PG_REGISTER_WITH_RESET_TEMPLATE(pidConfig_t, pidConfig, PG_PID_CONFIG, 3);
 
-#if !defined(DEFAULT_PID_PROCESS_DENOM)
-#if defined(STM32F411xE)
+#if !defined(DEFAULT_PID_PROCESS_DENOM) && defined(USE_GYRO_DENOM_CHECK)
```

Comment:
If `DEFAULT_PID_PROCESS_DENOM` is not defined and `USE_GYRO_DENOM_CHECK` is also not defined, we will not get a default pid process denominator at all. Is this intended?